src/main/java/eu/darkbot/api/objects/Location.java

```diff
@@ -4,42 +4,49 @@
     /**
      * Returns distance to location ox & oy
      */
-    default double distanceTo(double ox, double oy) {
-        ox -= getX();
-        oy -= getY();
-        return Math.sqrt(ox * ox + oy * oy);
-    }
+    double distanceTo(double ox, double oy);
 
     default double distanceTo(Locatable other) {
         return distanceTo(other.getX(), other.getY());
     }
 
-    default double angleTo(double ox, double oy) {
-        return Math.atan2(getY() - oy, getX() - ox);
-    }
+    /**
+     * @return angle to other location as radians.
+     */
+    double angleTo(double ox, double oy);
```

Comment:
same here, this is a pretty generic operation it can stay as a default, it is not making any extra assumptions about implementation